pycue/opencue/wrappers/service.py

```diff
@@ -172,3 +172,26 @@ def setTags(self, tags):
         :type: list<string>
         :param: list of tags to set"""
         self.data.tags[:] = tags
+
+
+class ServiceOverride(object):
+    def __init__(self, serviceOverride=None):
+        if serviceOverride:
+            self.id = serviceOverride.id
+            self.data = serviceOverride.data or service_pb2.Service().data
+        else:
+            defaultServiceOverride = service_pb2.ServiceOverride()
+            self.id = defaultServiceOverride.id
+            self.data = defaultServiceOverride.data
+        
+        self.stub = Cuebot.getStub("serviceOverride")
+
+    def delete(self):
+        self.stub.Delete(
+            service_pb2.ServiceOverrideDeleteRequest(service=self.data),
+            timeout=Cuebot.Timeout)
+    
+    def update(self):
```

Comment:
I feel like `commit` might be a more obvious name here -- when I see `update` I'm not sure whether that means "read values from the database and update the local instance" or "push local values to the database".

Reply:
I agree, but changing this now would cause impact on internal scrips that have already been implemented with `update`.  Would you be okay with keeping as it is if we add a doc explaining what it does?

```suggestion
    def update(self):
        """Commit a ServiceOverride change to the database"""
```